base/bootstrap.go

```diff
@@ -191,6 +194,24 @@ func (cc *CouchbaseCluster) GetConfig(location, groupID string, valuePtr interfa
 	return cc.configPersistence.loadConfig(b.DefaultCollection(), PersistentConfigPrefix+groupID, valuePtr)
 }
 
+func (cc *CouchbaseCluster) HasPost30Config(bucket string) (bool, error) {
+	if cc == nil {
+		return false, errors.New("nil CouchbaseCluster")
+	}
+
+	b, teardown, err := cc.getBucket(bucket)
+
+	if err != nil {
+		return false, err
+	}
+
+	defer teardown()
+
+	var valuePtr interface{}
+	_, err = cc.configPersistence.loadConfig(b.DefaultCollection(), "_sync:registry", valuePtr)
```

Comment:
Would be helpful to define

```
	SGRegistryKey                        = SyncPrefix + "registry"          // Registry of all SG databases defined for the bucket (for all group IDs)
```
in base/constants.go.  If I'm looking for interaction with the registry in future I might look there first.